openapi.yaml

```diff
@@ -850,11 +886,18 @@ definitions:
         type: string
         description: the unique ID of the resource
         example: abc123
+      meta:
+        properties:
+          description:
+            type: string
+            description: information about the way the ingredient is used in the pizza
+            example: only put pepperoni on half of the pizza
+
```

Comment:
This is the only way I can find to embed information like quantity of an ingredient in the ingredient relationship itself.